README.md

```diff
@@ -8,8 +8,8 @@ This toolkit provides methods to execute the TPC-H, TPC-DS, and SSB benchmarks o
 
 Important note: in order to guarantee compatibility between S64 DA and
 s64da-benchmark-toolkit, please check out the Git tag that corresponds to your
-version of S64 DA. For example, if your version of S64 DA is 5.1.0, clone this
-repository and run `git checkout v5.1.0` in the the repository’s root folder
+version of S64 DA. For example, if your version of S64 DA is X.X.X, clone this
```

Comment:
This is an example, so IMHO it should be a concrete number (like it was before)

Reply:
If we are ok having an old version as example I am ok too. But it feels (not so professional) to use an older version as an example, to me looks like its forgotten.

Reply:
Anyway since this text encourages people to checkout to an older branch, then I am ok